### DISCUSSION TITLE
Why Pade approximant order have been restricted?

### DISCUSSION THREAD

**Discussion by s-celles** [JuliaControl/ControlSystems.jl#921] [General]
Hello,

it's not very important but I wonder why Pade approximant order have been restricted?

https://github.com/JuliaControl/ControlSystems.jl/blob/0d825400f1fd0126a4ccaa63408ba2b38351d47a/lib/ControlSystemsBase/src/delay_systems.jl#L128-L137

Any idea?

**Comment by baggepinnen** [comment]
Because we do not have the coefficients for higher orders defined. Higher-order transfer functions are also numerically ill-conditioned, so you're likely to encounter some issues if you try to go much higher
